From c7a2f12ec0ee9a7bff07d12c50063c08e0dc7c75 Mon Sep 17 00:00:00 2001
From: crazy <crazy@frugalware.org>
Date: Mon, 6 Jun 2016 22:44:41 +0200
Subject: [PATCH 2/2] makepkg tell what dirs/files are broken * vmiklos told me
 to fix it and I fixed it in my local tree but forgot to push :| * now makepkg
 will tell what dir is wrong eg: * /usr/man should be /usr/share/man and so
 on..

---
 scripts/makepkg | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/scripts/makepkg b/scripts/makepkg
index bac0fdc..8ce169a 100755
--- a/scripts/makepkg
+++ b/scripts/makepkg
@@ -1733,13 +1733,14 @@ do
 		exit 1
 	fi
 	## broken /usr/share
-	if [ -d ${dir}/usr/man ] || [ -d ${dir}/usr/docs ] || [ -d ${dir}/usr/$pkgname ]; then
-		error "Broken package detected files are installed in /usr"
-		error "but should be /usr/share"
-		error "datadir may be broken!"
-		error "Bailing out, please fix your package!"
-	exit 1
-	fi
+	for file in man docs $pkgname; do
+		if [ -d ${dir}/usr/$file ]; then
+			error "Broken package detected files are installed"
+			error "in /usr/$file but should be /usr/share/$file"
+			error "Bailing out, please fix your package!"
+			exit 1
+		fi
+	done
 	## wrong unit path for systemd
 	if [ -d ${dir}/usr/lib/systemd/system ]; then
 		error "Your package has broken PATH for systemd unit files"
-- 
2.8.3

